frontend/src/metabase/redux/uploads.ts

```diff
@@ -42,6 +42,7 @@ export interface UploadFileProps {
   collectionId?: CollectionId;
   tableId?: TableId;
   modelId?: CardId;
+  uploadMode: "append" | "create" | "replace";
```

Comment:
Can we make this an enum and reference it across files? Stringly typed APIs can be tough to figure out why/where which values belong to what. While typesafe, it makes refactors harder as I'm searching for every reference to "append" in our codebase rather than everywhere an enum is imported.